LandSandBoat/server
# Retail accurate Palborough Mines elevator

**_I affirm:_**

## What does this pull request do?

This adds some functionality and changes some scripts to accurately represent the elevators & levers in Palborough Mines.

## Steps to test these changes

Use a lever for the Palborough Mines elevator, get an event with a Y/N dialog, click no and it does nothing.
Click the lever and say yes, it waits about 3 seconds and then moves. During the time it is moving or waiting you should not be able to trigger the dialogue.
If another player has this dialogue open it should do nothing even during the inactivity window.

The levers should only move if the elevator is going to activate. The states are independent of eachother.
